src/__integ__/__snapshots__/themes.test.ts.snap

```diff
@@ -4895,6 +4918,7 @@ Object {
   "motion-duration-transition-quick": "90ms",
   "motion-duration-transition-show-paced": "180ms",
   "motion-duration-transition-show-quick": "90ms",
+  "motion-easing-avatar-gen-ai-gradient": "cubic-bezier(0.7, 0, 0.3, 1)",
```

Comment:
maybe a question for visual design, is this _so_ much different from the existing curve that it is useful to create a new token?

```
"motion-easing-expressive": "cubic-bezier(0.84, 0, 0.16, 1)",
```

Reply:
During the early motion design exploration for this topic, I actually tried out `motion-easing-expressive`, however we decided not to use this since it's too intense for avatar gradient motion. So I'd like to stick to the `"motion-easing-avatar-gen-ai-gradient": "cubic-bezier(0.7, 0, 0.3, 1)",`